statsd/statsd_test.go

```diff
@@ -1022,3 +1022,46 @@ func (c *Client) formatV2(name string, value float64, tags []string, rate float6
 
 	return buf.String()
 }
+
+func TestEntityID(t *testing.T) {
+	envName := "DD_ENTITY_ID"
+	initialValue, initiallySet := os.LookupEnv(envName)
+	if initiallySet {
+		defer os.Setenv(envName, initialValue)
+	} else {
+		defer os.Unsetenv(envName)
+	}
+
+	// Set to a valid value
+	os.Setenv(envName, "testing")
+	client, err := New("localhost:8125")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(client.Tags) != 1 {
+		t.Errorf("Expecting one tag, got %d", len(client.Tags))
+	}
+	if client.Tags[0] != "_dd.entity_id:testing" {
```

Comment:
You can reuse `entityIDTagName`

Reply:
I like to "hardcode" what's user facing to ensure we don't introduce a functionnal regression by changing a constant as a side-effect of botched PR